src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManagerInterface.php

```diff
@@ -46,7 +46,7 @@ public function getPermissions($type, $identifier);
      *
      * @return array
      */
-    public function getUserPermissions(SecurityCondition $securityCondition, UserInterface $user);
+    public function getUserPermissions(SecurityCondition $securityCondition, $user);
```

Comment:
Same as above

Reply:
No, isn't possible because its a string when the user isn't logged in